pallets/subtensor/src/macros/dispatches.rs

```diff
@@ -901,11 +901,8 @@ mod dispatches {
         #[pallet::weight((Weight::from_parts(157_000_000, 0)
 		.saturating_add(T::DbWeight::get().reads(16))
 		.saturating_add(T::DbWeight::get().writes(30)), DispatchClass::Operational, Pays::No))]
-        pub fn register_network(
-            origin: OriginFor<T>,
-            identity: Option<SubnetIdentityOf>,
-        ) -> DispatchResult {
-            Self::user_add_network(origin, identity)
+        pub fn register_network(origin: OriginFor<T>) -> DispatchResult {
+            Self::user_add_network(origin)
         }
```

Comment:
I suggest we put the contents of `user_add_network` inline here.

Reply:
This isn't an effort to refactor the code base , and would rather stick to the existing pattern , where we separate the dispatches and the actual methods